CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Style guide
+
+* 4 space indentation
+
+## Pragmas
+
+* One pragma on each line
+* No alignment
+* Sort lexicographically
+
+```haskell
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE DataKinds #-}
+```
+
+## Module declaration and exports
+
+* Opening paren on its own line
+* Prefix with commas
+* End with `) where`
+
+```haskell
+module Data.Aeson.Types.ToJSON
+    (
+    -- * Core JSON classes
+      ToJSON(..)
+    -- * Liftings to unary and binary type constructors
+    , ToJSON1(..)
+    , toJSON1
+    ) where
+```
+
+## Imports
+
+* No alignment of import lists or qualifications
```

Comment:
let's see if we can teach `stylish-haskell` to adhere to these (and pragma) rules.